### PR TITLE
feat(.cspell.json):  add words

### PR DESCRIPTION
**Pull request by Shin-kyoto**
I want to add these words.

# words

|  word  | definition |
| ----- | --- |
| bbox | abbreviation of bounding box. example: https://github.com/tianweiy/CenterPoint/tree/master/det3d/core/bbox |
| bboxes | the plural form of bbox |
| PointPillars | [3D detection model proposed in CVPR 2019](https://openaccess.thecvf.com/content_CVPR_2019/papers/Lang_PointPillars_Fast_Encoders_for_Object_Detection_From_Point_Clouds_CVPR_2019_paper.pdf) |

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "**/log/**",
     "**/package-lock.json",
     "**/public/**",
-    "**/reports*/**"
+    "**/reports*/**",
+    "perception/bytetrack/lib/*"
```
Comment (kenji-miyake) [inline .cspell.json:42]: I think this shouldn't be included because it's a repository-specific configuration.

```suggestion
```

Comment (Shin-kyoto) [reply .cspell.json]: Thank you for tell me that!
I remove path to bytetrack's lib from ignorePaths

File: .cspell.json [hunk 2]
```diff
@@ -1231,6 +1235,7 @@
     "pointcloud",
     "pointclouds",
     "pointpainting",
+    "PointPillars",
```
Comment (kenji-miyake) [inline .cspell.json:1238]: This is invalid.

```suggestion
```

Comment (kenji-miyake) [reply .cspell.json:1238]: Did you intend this?

```suggestion
    "pointpillars",
```

**Comment by kenji-miyake** [comment]
@Shin-kyoto Could you consider adding word-by-word so that we can judge faster? :bow: 
Using https://github.com/tier4/autoware-spell-check-dict/actions/workflows/add-word.yaml would be your help.

File: .cspell.json (hunk 2 above)
Comment (HansRobo) [reply .cspell.json:1237]: To clarify Miyake-san's point, i will add some explanation, for @Shin-kyoto .
`cspell` recognize capital letter as an word splitter, in CamelCase.
So, `PointPillars` will be split into `Point` and `Pillars`, and they will make no errors by cspell.
This is a reason Miyake-san said "This is invalid".

And, as Miyake-san suggested, "pointpillars" may be all you want.

Comment (Shin-kyoto) [reply .cspell.json:1237]: Thank you for telling me that!
So I'll add `pointpillars` by workflow.

**Comment by Shin-kyoto** [comment]
> @Shin-kyoto Could you consider adding word-by-word so that we can judge faster? bow Using https://github.com/tier4/autoware-spell-check-dict/actions/workflows/add-word.yaml would be your help.

Thank you for telling me that!
So I'll add words by workflow.

**Comment by Shin-kyoto** [comment]
Close this PR because I'll add words by workflow.

**Pull request closed without merging**